lib/backends/webgl/glsl-coordinate-lib.ts

```diff
@@ -54,6 +71,1072 @@ export class CoordsGlslLib extends GlslLib {
       `)
     };
   }
+
+  /**
+   * Generates code for output sampler.
+   */
+
+  protected getOutputSamplingSnippet(): {[name: string]: GlslLibRoutine;} {
+    const outputLayout = this.context.programInfo.outputLayout;
+    if (!outputLayout.isPacked) {
```

Comment:
nit: swap two statements to check isPacked first?

Reply:
Thanks, updated

---
In reply to: [586188902](https://github.com/microsoft/onnxjs/pull/266#discussion_r586188902) [](ancestors = 586188902)